analyzers/src/SonarAnalyzer.CSharp/Rules/LossOfFractionInDivision.cs

```diff
@@ -106,6 +110,33 @@ private static bool TryGetTypeFromAssignmentToFloatType(SyntaxNode division, Sem
                 return type.IsAny(KnownType.NonIntegralNumbers);
             }
 
+            if (division.Ancestors().LastOrDefault(x => TupleExpressionSyntaxWrapper.IsInstance(x)) != null)
+            {
+                if (division.Ancestors().OfType<AssignmentExpressionSyntax>().FirstOrDefault() is { } assignmentSyntax
+                    && assignmentSyntax.MapAssignmentArguments() is { } assignmentMappings)
+                {
+                    var assignementLeft = assignmentMappings.Where(x => x.Right.Equals(division)).FirstOrDefault().Left;
+                    if (assignementLeft != null)
+                    {
+                        type = semanticModel.GetTypeInfo(assignementLeft).Type;
+                        return type.IsAny(KnownType.NonIntegralNumbers);
+                    }
+                }
+                else if (division.Ancestors().OfType<VariableDeclarationSyntax>().FirstOrDefault() is { } variableDeclaration)
```

Comment:
This should be
`if division is 
{ Parent: EqualsValueClauseSyntax { Parent: VariableDeclaratorSyntax { Parent: VariableDeclarationSyntax variableDeclaration variableDeclaration} } })`

Reply:
I made a slightly different change here. Could you please re-review this part?

Reply:
Got it! 
Thanks 😄